lib/task-data/tasks/bootstrap-rancher.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Bootstrap Rancher',
+    injectableName: 'Task.Linux.Bootstrap.Rancher',
+    implementsTask: 'Task.Base.Linux.Bootstrap',
+    options: {
+        kernelFile: 'vmlinuz',
+        initrdFile: 'initrd',
+        kernelUri: '{{ api.server }}/common/{{ options.kernelFile }}',
```

Comment:
Can we maybe store the rancher kernel and initrd with more descriptive file names if we're putting them in /common?